cmd/cerbos/compile/compile.go

```diff
@@ -60,26 +43,28 @@ cerbos compile --skip-tests /path/to/policy/repo
 `
 
 type Cmd struct {
-	Dir           string `help:"Policy directory" arg:"" required:"" type:"existingdir"`
-	Format        string `help:"Output format (${enum})" default:"pretty" enum:"pretty,plain,json" short:"f"`
-	Tests         string `help:"Path to the directory containing tests. Defaults to policy directory." type:"existingdir"`
-	RunRegex      string `help:"Run only tests that match this regex" name:"run"`
-	SkipTests     bool   `help:"Skip tests"`
-	IgnoreSchemas bool   `help:"Ignore schemas during compilation"`
-	Verbose       bool   `help:"Verbose output on test failure"`
+	Dir           string               `help:"Policy directory" arg:"" required:"" type:"existingdir"`
+	Output        flagset.OutputFormat `help:"Output format (${enum})" default:"tree" enum:"tree,pretty,json" short:"o"`
+	Tests         string               `help:"Path to the directory containing tests. Defaults to policy directory." type:"existingdir"`
+	RunRegex      string               `help:"Run only tests that match this regex" name:"run"`
+	SkipTests     bool                 `help:"Skip tests"`
+	IgnoreSchemas bool                 `help:"Ignore schemas during compilation"`
+	Verbose       bool                 `help:"Verbose output on test failure"`
+	Color         bool                 `help:"Enable or disable colored output" default:"true"`
```

Comment:
Since colour is on by default, I think we should name this `no-color`. It's more natural because boolean flags are "set" by  just adding them to the command (`cerbos compile --no-color` is better than `cerbos compile --color=false`)

Since there's a dedicated flag to turn colour off, maybe the default JSON output should coloured as well like we do in the `audit` output: https://github.com/cerbos/cerbos/blob/4f04ba056aa176c8daa4ce3166a2388ac2c5666c/cmd/cerbosctl/audit/audit.go#L162